ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```diff
@@ -61,5 +61,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-openshift-tests run-monitor ${MONITOR_ARGS:-} --artifact-dir $STORE_PATH > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
+DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
```

Comment:
Please add `incluster-disruption-serializer` as well, https://github.com/openshift/origin/pull/28277 would add that